wf-galleon/wildfly_channel_in_zips.adoc

```diff
@@ -0,0 +1,125 @@
+---
+categories:
+- provisioning
+---
+= [Experimental] Update WildFly download distributions using channels
+:author:            Jeff Mesnil
+:email:             jmesnil@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+[abstract]
+As an user, I can download WildFly from a zip archive and be able to update it to a more recent release.
+
+== Overview
+
+WildFly now publishes https://repo1.maven.org/maven2/org/wildfly/channels/[channels] that is a mechanism to keep a WildFly installation up to date with the latest releases.
+
+This works out of the box if users is provisioning WildFly (using the wildfly-maven-plugin or Prospero).
+However for users that are downloading WildFly from our traditional distributions (zip & tgz archives) they can not take advantage of this mechanism.
+
+If our archives would work out of the box with channels, it would significantely simplify updating an installation.
+
+The workflow would be as simple as:
+
+1. Download WildFly version x.y.z from our download page
+2. Use it as normal
+3. A micro release of WildFly x.y.(z+ 1) is made available
+4. Run a script in the existing WildFly installation to update if from x.y.z to x.y.(z+1)
```

Comment:
Does that mean we would include Prospero in the WildFly zip? An alternative could be using standalone prospero tool to perform the update

Reply:
This is an open question. There is a `bin/installation-manager.sh` that does work (and I've no idea what its intent)

I'm not keen on adding another tool inside WildFly distributions and I wonder if we should make the update mechanism accessible through `./bin/jboss-cli.sh`...

Reply:
(prospero lib would still be needed as a JBoss module but not as something surfacing to the UI)

Reply:
I updated the proposal to reuse the  `installation-manager` script to update WildFly and mention that we would use prospero as lib